src/Aspire.Dashboard/Components/Pages/Traces.razor

```diff
@@ -14,9 +13,11 @@
 
 <PageTitle><ApplicationName ResourceName="@nameof(Dashboard.Resources.Traces.TracesPageTitle)" Loc="@Loc" /></PageTitle>
 
-<div class="traces-layout">
-    <h1 class="page-header">@Loc[nameof(Dashboard.Resources.Traces.TracesHeader)]</h1>
-    <FluentToolbar Orientation="Orientation.Horizontal">
+<AspirePageContentLayout AddNewlineOnDesktopToolbar="true">
+    <PageTitleSection>
+        <h1 class="page-header">@Loc[nameof(Dashboard.Resources.Traces.TracesHeader)]</h1>
+    </PageTitleSection>
+    <ToolbarSection>
```

Comment:
Filter text is lost when switching between views.

Reply:
This has been fixed